lib/router.js

```diff
@@ -70,10 +70,18 @@ Router.prototype._loadModule = function (modDef, symbols) {
         default:
             throw new Error('unknown module type ' + modDef.type + ' (for module ' + modDef.name + ').');
     }
+    // append the log property to module options, if it is not present
+    if (modDef.options) {
+        modDef.options.log = modDef.options.log || this._options.log;
+    }
     // let the error propagate in case the module cannot be loaded
     var modObj = require(loadPath)(modDef.options);
     this._modules.set(modDef, modObj);
     for (var symbol in modObj) {
+        if (!modObj.hasOwnProperty(symbol)) {
```

Comment:
Maybe we should do `Object.keys().forEach()` instead. Saves the `hasOwnProperty` check & is a bit clearer.